### PR TITLE
fix: moved buildJson to constructor

### PR DESCRIPTION
**Pull request by athul-v-nair-01**
moved buildJson functionality to constructor and corrected paths for the packages

### CODE REVIEW

File: src/HubspotSchema.php [hunk 1]
```diff
@@ -57,19 +48,17 @@ public function buildJson(): array
                         $builder->addRecordType($recordType);
                     }
                 } else {
-                    return ["Empty CRM object properties"];
+                    print_r("Empty CRM object properties");
```
Comment (arun-experion) [inline src/HubspotSchema.php:51]: Correct this

File: src/HubspotSchema.php [hunk 2]
```diff
@@ -57,19 +48,17 @@ public function buildJson(): array
...
             // DiscoverResult.json will contain the data in required JSON format
-            file_put_contents(__DIR__ . '/../DiscoverResult.json', json_encode($arrayJson, JSON_PRETTY_PRINT));
-    
-            return $arrayJson;
+            file_put_contents(__DIR__ . '/../DiscoverResult.json', json_encode(json_decode($jsonStructure, true), JSON_PRETTY_PRINT));
+
+            parent::__construct($builder->toArray());
         } else {
-            return ["No CRM objects Found"];
+            print_r("No CRM objects Found");
```
Comment (arun-experion) [inline src/HubspotSchema.php:61]: Correct this

**Review by arun-experion — COMMENTED** [review]
Review comments added

File: src/HubspotSchema.php (hunk 1 above)
Comment (athul-v-nair-01) [reply src/HubspotSchema.php:51]: Since it is a constructor we cant return right?

**Pull request merged**